client/src/Data/Api.elm

```diff
@@ -2,7 +2,7 @@ module Data.Api exposing
     ( Endpoint(..), get, post, put, patch, delete
     , withJsonBody, withStringResponse, withJsonResponse, withStringErrorHandler, withErrorHandler
     , sendRequest
```

Comment:
I think we should add some loading state when the user click the suggested address:

I pushed the change here, maybe you can check my commit:
- https://github.com/Southern-Exposure-Seed-Exchange/southernexposure.com/commit/e4c5e0440a6e1fc1e576f5440c172571ef3a53ab

[Screencast from 08-29-2025 07:33:42 AM.webm](https://github.com/user-attachments/assets/03b14b11-c046-4d3b-acda-f5db439b6c07)

Reply:
Thanks, cherry-picked to my branch